core/base-service/base.js

```diff
@@ -1,8 +1,12 @@
 'use strict'
 
 // See available emoji at http://emoji.muan.co/
```

Comment:
Admittedly a very minor nit 😄, but this comment should probably be above the `emojic` line right?